storage/src/robocorp/storage/__init__.py

```diff
@@ -0,0 +1,173 @@
+import logging
+import random
+import time
+from typing import Dict, List, Optional
+
+from ._requests import Requests, RequestsHTTPError
+from ._storage import AssetNotFound
+from ._storage import get_assets_client as _get_assets_client
+from ._utils import with_lazy_objects as _with_lazy_objects
+
+__version__ = "0.1.0"
+version_info = [int(x) for x in __version__.split(".")]
+
+LOGGER = logging.getLogger(__name__)
+_with_lazy_assets = _with_lazy_objects(_assets_client=_get_assets_client)
+
+
+@_with_lazy_assets
+def list_assets(*, _assets_client: Requests) -> List[Dict]:
+    """List all the existing assets.
+
+    Returns:
+        A list of assets where each asset is a dictionary with fields like 'id' and
```

Comment:
Is the shape of the asset defined by the API, or can it be any data that is user-defined? If the former, I'd prefer these to be properly typed, but that can be left as later improvements. Especially if the API is subject to change still.

Reply:
Yes, defined by the API and quite stable and easy to type and what to expect, good to be done right from this PR.